src/human_player_module.h

```diff
@@ -0,0 +1,17 @@
+#include "cards_management.h"
```

Comment:
Let's include header guards in the header file. eg:

#ifndef __HUMAN_PLAYER_HEADER__
#define __HUMAN_PLAYER_HEADER__
...
#endif // __HUMAN_PLAYER_HEADER__

Reply:
Done